src/components/views/settings/index.tsx

```diff
@@ -163,7 +178,21 @@ export const Settings: React.FC<SettingsProps> = ({navigation}) => {
                   accessibilityLabel={label}
                   accessibilityRole="button"
                   accessibilityHint={hint}
-                  onPress={() => navigation.navigate(screen)}>
+                  onPress={() =>
+                    screen === 'test'
+                      ? Linking.openURL(
```

Comment:
Two related things here about this section:

- On Android / Samsung Mail all the whitespace indentation on the left inside the \`\` ends up as part of the email body. It's okay (and usually necessary) for content in \`\` to ignore the indentation of the surrounding code
- For code readability, since this is a big function it'd be good to pull it out of the render return value and just reference the variable here in the `onPress`